ui/main/package.json

```diff
@@ -27,9 +27,10 @@
     "@fortawesome/fontawesome-free": "6.2.0",
     "@fullcalendar/angular": "5.11.2",
     "@fullcalendar/bootstrap": "5.11.3",
-    "@fullcalendar/core": "5.11.3",
-    "@fullcalendar/daygrid": "5.11.3",
+    "@fullcalendar/core": "^5.11.3",
+    "@fullcalendar/daygrid": "^5.11.3",
     "@fullcalendar/interaction": "5.11.3",
+    "@fullcalendar/rrule": "^5.11.3",
```

Comment:
Do not use ^ for version number as it does guarantee the version (we have to keep upgrades under control wuht precise version of dependencies)

Reply:
Done.